knip.js

```diff
@@ -0,0 +1,104 @@
+// @ts-check
+const testEntry = 'test/**/*.test.js';
+
+/** @type {import('knip').KnipConfig} */
+export default {
+	ignore: ['**/test/**/{fixtures,_temp-fixtures}/**', '.github/scripts/**'],
+	tags: ['-lintignore'],
+	ignoreWorkspaces: [
+		'examples/**',
+		'**/{test,e2e}/**/{fixtures,_temp-fixtures}/**',
+		'benchmark/**',
+	],
+	workspaces: {
+		'.': {
+			ignoreDependencies: ['@astrojs/check'],
+			ignoreBinaries: ['docgen', 'docgen:errors', 'playwright'],
+		},
+		'packages/astro': {
+			entry: [
+				// Can't be detected automatically since it's only in package.json#files
+				'templates/**/*',
+				testEntry,
+				'test/types/**/*',
+				'e2e/**/*.test.js',
+				'test/units/teardown.js',
+			],
+			ignore: ['**/e2e/**/{fixtures,_temp-fixtures}/**', 'performance/**/*'],
+			ignoreDependencies: [
+				'rehype-autolink-headings',
+				'rehype-slug',
+				'rehype-toc',
+				'remark-code-titles',
+			],
+		},
+		'packages/astro-prism': {},
+		'packages/astro-rss': {
+			entry: [testEntry],
```

Comment:
Isn't there a way to reduce repetition? I am afraid that if we add a new package, we will forget to update file